pkg/plugins/runtime/k8s/webhooks/validation.go

```diff
@@ -138,6 +146,15 @@ func syncErrorResponse(resType core_model.ResourceType, cpMode core.CpMode) admi
 	}
 }
 
+func isKumaServiceAccount(userInfo authenticationv1.UserInfo) bool {
+	elms := strings.Split(userInfo.Username, ":")
+	// system:serviceaccount:<namespace>:kuma-control-plane
+	if len(elms) == 4 && elms[3] == "kuma-control-plane" {
```

Comment:
This has to work with other products based on Kuma, so `kuma-control-plane` won't work. Also, double-check if the value changes with HELM.

Maybe what we could do is to only check the namespace?